File: hw/vizcaino/u2/hw14MongoDb/hw14MongoDbJava/hw14Interface/src/main/java/ec/edu/espe/model/MongoConnection.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.model;

/**
 *
 * @author Adrian Vizcaino <The-Softwarriors at ESPE>
 */
import com.mongodb.client.MongoClient;
import com.mongodb.client.MongoClients;
import com.mongodb.client.MongoDatabase;

public class MongoConnection {

    private static MongoDatabase database;

    static {

        String uri =
        "mongodb+srv://Adrian:Adrian@cluster0.e4n4vbs.mongodb.net/?appName=Cluster0";

        MongoClient client =
                MongoClients.create(uri);

        database =
                client.getDatabase("InventoryDB");
    }

    public static MongoDatabase getDatabase() {
        return database;
    }
}

File: hw/vizcaino/u2/hw14MongoDb/hw14MongoDBJS/config/mongoConnection.js
const dns = require("dns");

dns.setServers(["8.8.8.8", "8.8.4.4"]);

const { MongoClient } = require("mongodb");

const uri =
"mongodb+srv://Adrian:Adrian@cluster0.e4n4vbs.mongodb.net/?appName=Cluster0";

const client = new MongoClient(uri);

async function getDatabase() {

    await client.connect();

    return client.db("hw14MongoDBJS");
}

module.exports = { getDatabase };